optimum/exporters/openvino/__main__.py

```diff
@@ -436,6 +431,8 @@ class StoreAttr(object):
         if quantization_config is None:
             continue
 
+        submodel = core.read_model(submodel_path)
```

Comment:
Shouldn't this be moved after the following if? Otherwise model will be read even if nncf is not installed